asn1crypto/x509.py

```diff
@@ -947,8 +947,10 @@ def native(self):
                 for type_val in rdn:
                     field_name = type_val['type']
                     if field_name in self._native:
-                        self._native[field_name] = [self._native[field_name]]
-                        self._native[field_name].append(type_val['value'])
+                        existing = self._native[field_name]
+                        if not isinstance(existing, list):
+                            existing = self._native[field_name] = [existing]
+                        existing.append(type_val['value'])
                     else:
```

Comment:
I believe you are correct in that `type['val']` itself won't be a `list`.

Do you have an example certificate that you found this bug with that you could add to the fixtures so we could add a test to ensure the behavior stays the same in the future?